certified-connectors/KyndrylMainframe/apiDefinition.swagger.json

```diff
@@ -0,0 +1,707 @@
+        "summary": "Get policy"
+      }
+    },
+    "/customernumber/Custdetailadd": {
+      "post": {
+        "tags": [
+          "customernumber"
+        ],
+        "operationId": "postCustomerdetailsupd",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "postCustomerdetailsupd_request",
+            "description": "request body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/postCustomerdetailsupd_request"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/postCustomerdetailsupd_response_200"
+            }
+          }
+        },
+        "summary": "Post customer details"
+      }
+    },
+    "/customernumber/Custdetailupd/{num}/{firstname}": {
+      "put": {
+        "tags": [
+          "customernumber"
+        ],
+        "operationId": "putCustomerdetail",
+        "parameters": [
+          {
+            "name": "num",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "firstname",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "body",
+            "required": true,
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/putCustomerdetail_request"
+            },
+            "description": "request body"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/putCustomerdetail_response_200"
+            },
+            "examples": {
+              "application/json": {
+                "LGCMAREA": {
+                  "CA_REQUEST_ID": "string",
+                  "CA_RETURN_CODE": 0,
+                  "CA_CUSTOMER_NUM": 0,
+                  "CA_REQUEST_SPECIFIC": "string",
+                  "CA_CUSTOMER_REQUEST": {
+                    "CA_FIRST_NAME": "string",
+                    "CA_LAST_NAME": "string",
+                    "CA_DOB": "string",
+                    "CA_HOUSE_NAME": "string",
+                    "CA_HOUSE_NUM": "string",
+                    "CA_POSTCODE": "string",
+                    "CA_NUM_POLICIES": 0,
+                    "CA_PHONE_MOBILE": "string",
+                    "CA_PHONE_HOME": "string",
+                    "CA_EMAIL_ADDRESS": "string",
+                    "CA_POLICY_DATA": "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Put customer details"
+      }
+    },
+    "/customernumber/custnum/{num}": {
+      "get": {
+        "tags": [
+          "customernumber"
+        ],
+        "operationId": "getCustomerdetail",
+        "parameters": [
+          {
+            "name": "num",
```

Comment:
@RTkyndryl for all parameters The 'description' property is required and the 'x-ms-summary' property is required.
Please add both for each parameter in the swagger file.